src/themes/uclouvain/app/entity-groups/person-entity/item-pages/person-page.component.html

```diff
@@ -8,20 +8,9 @@
 <div class="container my-3 p-0">
   <article>
     <!-- HEADER (Type + name) -->
-    <div class="mb-5 row">
-      <div class="mr-4">
-        <ds-metadata-field-wrapper [hideIfNoTextContent]="false">
-          <ds-thumbnail [thumbnail]="object?.thumbnail | async"
-                        [defaultImage]="'assets/images/person-placeholder.svg'"
-                        [alt]="'thumbnail.person.alt'"
-                        [placeholder]="'thumbnail.person.placeholder'">
-          </ds-thumbnail>
-        </ds-metadata-field-wrapper>
-      </div>
```

Comment:
Maybe it could be better to add an `@Input showThumbnail: boolean = false` parameter to be more resilient with next rebase ?